packages/suite/src/views/wallet/nfts/NftsNavigation.tsx

```diff
@@ -0,0 +1,87 @@
+import { Dispatch, SetStateAction, useEffect, useState } from 'react';
+
+import { SelectedAccountLoaded } from '@suite-common/wallet-types';
+import { selectNftDefinitions } from '@suite-common/token-definitions';
+import { spacings } from '@trezor/theme';
+import { Row } from '@trezor/components';
+
+import { useSelector } from 'src/hooks/suite';
+import { NavigationItem } from 'src/components/suite/layouts/SuiteLayout/Sidebar/NavigationItem';
+import { getNfts } from 'src/utils/wallet/nftUtils';
+import { selectRouteName } from 'src/reducers/suite/routerReducer';
+import { SearchAction } from 'src/components/wallet/SearchAction';
+import { filterNftTokens } from '@suite-common/wallet-utils';
+
+interface NftsNavigationProps {
+    selectedAccount: SelectedAccountLoaded;
+    searchQuery: string;
+    setSearchQuery: Dispatch<SetStateAction<string>>;
+}
+
+export const NftsNavigation = ({
```

Comment:
Most of the logic is copy/paste of TokensNavigation, would it be possible to unify it?

Reply:
Yes,  I agree, I unified it